framework/web/Request.php

```diff
@@ -1444,8 +1419,8 @@ public function validateCsrfToken($token = null)
     /**
      * Validates CSRF token
      *
-     * @param string $token
-     * @param string $trueToken
+     * @param string $token The masked client-supplied token.
```

Comment:
I suggest we merge it instead; this part of the code hasn't changed during this review cycle.
At some point it starts to feel like hitting a moving target.

Regarding moving it to a `SecurityHelper`, that thought crossed my mind as well, especially since it could host a couple of function that are currently hosted by the security component as well.

On the other hand, changing or moving this later requires only a small fix from a developers perspective.

Maybe for 2.1 the security component could be more lean and more functionality can be moved to a helper, including this functionality?